### ISSUE TITLE
А что есть в схеме С1 и С2 

### ISSUE THREAD

**Issue by fogddragon** [AlexGyver/NixieClock#2]
В схемен и на плате шелкографией нанесены места под С1 и С2 ?что это? судя по  списку =компонентов это конденсаторы, какой нужны емкости?

**Comment by x3mEr** [comment]
C1 - должно быть, электролит (6.3V, 47-100 мкФ).

**Comment by technotrasher** [comment]
> 
> 
> C1 - должно быть, электролит (6.3V, 47-100 мкФ).

Неверно. исходя из схемы подключения LM7805 конденсаторы задаются так - https://ruselectronic.com/_files/200000850-1750b184b4/7805.JPG

**Comment by x3mEr** [comment]
> 
> 
> > C1 - должно быть, электролит (6.3V, 47-100 мкФ).
> 
> Неверно. исходя из схемы подключения LM7805 конденсаторы задаются так - https://ruselectronic.com/_files/200000850-1750b184b4/7805.JPG

Точно.
